stable/ceph-base/bundle.yaml

```diff
@@ -29,19 +16,23 @@ services:
     annotations:
       gui-x: '1000'
       gui-y: '500'
-    charm: cs:ceph-osd-294
-    comment: SET osd-devices to match your environment
+    charm: cs:ceph-osd-303
     num_units: 3
     options:
       osd-devices: /dev/sdb
-      source: cloud:bionic-train
+      source: distro
     to:
     - '0'
     - '1'
     - '2'
-  ntp:
```

Comment:
https://docs.openstack.org/project-deploy-guide/charm-deployment-guide/latest/install-openstack.html#ntp

I think we still need NTP. Why is this getting removed?

Reply:
For context, on Bionic the `ntp` charm deploys chrony.

https://jaas.ai/ntp